ccd-definition/CaseEventToFields/draftSDO.json

```diff
@@ -0,0 +1,171 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "draftSDO",
+    "CaseFieldID": "hearingDate",
```

Comment:
I remember seeings ${} when the hearing date is not present. Do we need to display this conditionally? What does Mia believe?

Reply:
Another problem we have got this on only one screen :(. I remember issues having the same field on multiple screens, maybe it's fine for labels? If not can you check with Design they are happy with having this only on one? If not probably we need to do some workarounds...

Reply:
I need to checkout how it looks when hearingDate is not populated.

Yes, there was an issue with displaying the field multiple times through the event, even as a label 🙄 is there a clever way to get around this? Creating a complex type of hearingDate + collection direction?

Reply:
Yeah maybe we could try to build this in the collection. Could you discuss this with Moray / Mia if could be done as separate story? I would be inclined to close this without making changes like that.

Reply:
Have messaged Moray and Mia and will suggets extracting this to a separate ticket 👍

Reply:
Moray is going to draft another ticket to capture this